Sage/carbon
# feat: Menu and NavigationBar enhancements - FE-4492

### Proposed behaviour

This PR adds 2 additional themes to Menu and NavigationBar - black and white

![Zrzut ekranu 2021-12-16 o 14 33 15](https://user-images.githubusercontent.com/20651022/146381399-a0c39fe2-67ee-4184-b659-843e205e6d92.png)


### Checklist




#### QA



### Testing instructions

Check existing Menu and NavigationBar stories to see if original themes have not been modified by mistake and confirm that the new themes are the same as in provided designs
